### DISCUSSION TITLE
Quest for Soroban NFT Entries

### DISCUSSION THREAD

**Discussion by secondbutz** [stellar/sorobanathon#155] [General]
Here my entries for the Sorobanathon

Prologue - The quest begins
https://litemint.com/items/GB3NTHCHIF3WB44XG3HFUISJWAHRQCXILM5DGMQFZWEAEAVKK7WWL66T/soroprol

Chapter One - https://litemint.com/items/GD3DOBFXT4SUQ2P2WDXROQEYA4F7YTS3NPZMXICF3JC3O3YG7FRVVOWO/sorochap1

Chapter Two - https://litemint.com/items/GCZC6Z4DYQFLO2EUGLGNRPH33AUUWX7FSBPZUTTLE744ISA3UJDMGRR4/sorochap2

Chapter Three - https://litemint.com/items/GAGBP5FKXGPX5GS5A6WICCBYXAHEJCEGX6BKLKJOSTT7RL6ZY4BFAUPG/sorochap3

Chapter Four - https://litemint.com/items/GA4YHDU7GWA5J7YNDPNRKVN6RGUUHK6KO2KKMLAO7OPPSQIFO6JZFXPM/sorochap4

Chapter Five - https://litemint.com/items/GBIYCAMFEYOT4HCZCYVWPDHGG3SMQZ5V2FTCV4VHEBP73LTYGLJHT3LC/sorochap5

Chapter Six - https://litemint.com/items/GASXUW57ZCOR3RFPNB2U7C7RFAJS5NNTJZFNRYD3HWJY3PQOKZLPEBBE/sorochap6

that was fun 

thx

**Comment by stellarjohn** [comment]
Congratulations! Your contribution to has been accepted! 🎉
Your submission may qualify for a reward of 5100 XLM! One last step: to receive that reward, please fill out this [form](https://stellarform.typeform.com/Sorobanathon) completely and accurately.

Please let us know if you have any questions.

Cheers!
John
